### PR TITLE
Database tests

### PR DESCRIPTION
**Pull request by mananwason**
@creativepsyco Please check and merge.

### CODE REVIEW

**Comment by mananwason** [comment]
@creativepsyco I squashed the commits.

File: app/src/androidTest/java/org/fossasia/openevent/DatabaseTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        DbHelper dbHelper = new DbHelper(mContext);
+        SQLiteDatabase db = dbHelper.getWritableDatabase();
+        assertTrue(db.isOpen());
+        db.close();
+    }
+
+    public void TestInsertData(){
+        DbHelper dbHelper = new DbHelper(mContext);
+        SQLiteDatabase db = dbHelper.getWritableDatabase();
+
```
Comment (creativepsyco) [inline app/src/androidTest/java/org/fossasia/openevent/DatabaseTest.java]: No assert here what are you testing?

Comment (mananwason) [inline app/src/androidTest/java/org/fossasia/openevent/DatabaseTest.java]: Yeah first I was writing all tests in this class itself but then I made different classes, so I'll just remove this.

**Comment by creativepsyco** [comment]
Can you rebase with fossasia:master and push again? The travis test won't work if the PR cannot be merged (or is in a merge-able) state

**Pull request closed without merging**